### ISSUE TITLE
Documentation not reloading automatically

### ISSUE THREAD

**Issue by tlacoyodefrijol** [okfn/opendatasurvey#973]
I pushed changes on the documentation and it hasn't reflected in census.okfn.org. I think it's the same issue as before.

**Comment by brew** [comment]
@tlacoyodefrijol Which PR does this relate to? What were the changes? RTD says it built within the last day.

**Comment by brew** [comment]
TL;DR: This seems to be working now. Please confirm at census.okfn.org and close the issue if happy.

I see an example of something that should be changed in the github repo. When github master has changed, RTD builds a new version, but it doesn't seem to change at census.okfn.org. I tried to manually trigger a new build, but again, it doesn't seem to get updated. I've poked around RTD settings and explicitly added the default branch to use when building ('master', although this should be the default RTD uses anyway, without having to enter it manually). After this the build seems to be deployed. Whether my settings change caused it to update or not, I don't know, so we'll have to continue to keep an eye on it, and confirm changes have been updated.
